Linkedlst.rb

```diff
@@ -0,0 +1,92 @@
+class Node
+
+    attr_accessor :value, :next_node
+
+    def initialize val,next_node
```

Comment:
If this is the reason, then please get rid of the next_node argument from the initializer.